apps/web/src/views/Swap/hooks/useFeeSaved.ts

```diff
@@ -0,0 +1,31 @@
+import { ChainId, Currency, CurrencyAmount } from '@pancakeswap/sdk'
+import { DAI, USDC, USDT } from '@pancakeswap/tokens'
+import { useMemo } from 'react'
+
+const MAJOR_DEFINED_STABLE_COINS = {
+  [ChainId.ETHEREUM]: [USDC[ChainId.ETHEREUM], DAI[ChainId.ETHEREUM], USDT[ChainId.ETHEREUM]],
+  [ChainId.ARBITRUM_ONE]: [USDC[ChainId.ARBITRUM_ONE]],
+  [ChainId.ARBITRUM_GOERLI]: [USDC[ChainId.ARBITRUM_GOERLI]],
+  [ChainId.POLYGON_ZKEVM]: [USDC[ChainId.POLYGON_ZKEVM], DAI[ChainId.POLYGON_ZKEVM]],
+  [ChainId.POLYGON_ZKEVM_TESTNET]: [USDC[ChainId.POLYGON_ZKEVM_TESTNET]],
+  [ChainId.BSC]: [USDC[ChainId.BSC]],
```

Comment:
how about `usdt`? I think swapping between `usdt` and `usdc` doesn't charge for interface fee as well?